seismicpro/src/survey.py

```diff
@@ -752,27 +777,27 @@ def _process_limits(self, limits):
             raise ValueError('Empty traces after setting limits.')
         return slice(*limits)
 
-    def remove_dead_traces(self, inplace=False, **kwargs):
+    def remove_dead_traces(self, inplace=False, bar=True):
         """ Removes dead (constant) traces from survey's data.
```

Comment:
```suggestion
        """ Remove dead (constant) traces from survey's data.
```
I'd also change `data` to something more appropriate, since `data` may be confused with trace data, which the survey does not store